workloads/kube-burner/touchstone-configs/containerMemory-avg.json

```diff
@@ -0,0 +1,21 @@
+{
+  "elasticsearch": {
+    "ripsaw-kube-burner": [
```

Comment:
Why 4 configuration files rather than a single one holding all queries?

Reply:
I sent a example on slack on how to consolidate this.